srcpkgs/virt-manager/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'virt-manager'
 pkgname=virt-manager
 version=4.1.0
-revision=3
+revision=4
```

Comment:
If all you're doing is fixing tests, there's no need for a revbump here.

Reply:
done